src/components/Spotlight/Spotlight.tsx

```diff
@@ -237,10 +289,10 @@ const Spotlight = ({
       lightPath: finalRectangularPath,
       circularLightPath: circularPath,
 
-      annotationTransform: `translate(${rect.x}px, ${rect.y}px) translate(0%, -100%)`,
+      annotationTransform: `translate(${rect.x}px, ${rect.y}px) translate(0%, -100%)`, // TODO: change second translate to make different attach positions
```

Comment:
@aVileBroker This looks great! Thanks for doing this :-)
I have a few questions about this comment. Is this something that's too big to include in this PR? Or perhaps does it just make more sense in another PR? And if so, is there another Jira ticket for this work? Cheers!

Reply:
This would be the other ticket, I can reference it in the comments
https://github.com/Headstorm/foundry-ui/issues/315